src/_tkagg.cpp

```diff
@@ -24,6 +24,9 @@
  * also.
  */
 #define WIN32_DLL
+static inline PyObject *PyErr_SetFromWindowsErr(int ierr) {
+    return PyErr_SetString(PyExc_OSError, "Call to EnumProcessModules failed");
```

Comment:
You can't `return PyErr_SetString(...)` as the latter returns void.  So it's probably rather `PyErr_SetString(...); return NULL;`.